src/material/autocomplete/autocomplete.scss

```diff
@@ -1,21 +1,30 @@
 @use '@angular/cdk';
-@use '@material/menu-surface/mixins' as mdc-menu-surface;
-@use '@material/list/evolution-mixins' as mdc-list;
-
-@include mdc-menu-surface.core-styles($query: structure);
-
-// Note that the `.mdc-menu-surface` is here in order to bump up the specificity
-// and avoid interference with `mat-menu` which uses the same mixins from MDC.
-.mdc-menu-surface.mat-mdc-autocomplete-panel {
+@use '../core/style/elevation';
+@use '../core/tokens/token-utils';
+@use '../core/tokens/m2/mat/autocomplete' as tokens-mat-autocomplete;
+
+// Even though we don't use the MDC styles, we need to keep the classes in the
+// DOM for the Gmat versions to work. We need to bump up the specificity here
+// so that it's higher than MDC's styles.
+div.mat-mdc-autocomplete-panel {
+  @include elevation.elevation(8);
   width: 100%; // Ensures that the panel matches the overlay width.
   max-height: 256px; // Prevents lists with a lot of option from growing too high.
-  position: static; // MDC uses `absolute` by default which will throw off our positioning.
   visibility: hidden;
-  // MDC sets the transform-origin programmatically based on whether the dropdown is above or
-  // below the input. We use our own positioning logic, so we need to set this ourselves.
   transform-origin: center top;
+  overflow: auto;
+  padding: 8px 0;
+  border-radius: 4px;
```

Comment:
I can see the `border-radius` and `padding` potentially being themable here, but MDC's menu surface doesn't expose tokens for them so I didn't either. We can always add custom tokens afterwards.